packages/@aws-cdk/aws-certificatemanager/lib/util.ts

```diff
@@ -16,3 +19,32 @@ export function apexDomain(domainName: string): string {
   }
   return accumulated.reverse().join('.');
 }
+
+export const isDnsValidatedCertificate = (cert: ICertificate): cert is DnsValidatedCertificate =>
+  cert.hasOwnProperty('domainName');
+
+export const getCertificateRegion = (cert: ICertificate): string | undefined => {
```

Comment:
I don't see a good reason not to use the boring `function` syntax here

Reply:
Done. My default "boring" has become arrow functions over the years, sorry 😄